app/models/cdo/view/check-activities.js

```diff
@@ -0,0 +1,16 @@
+const { mapActivityDtoToCheckActivityRow } = require('../../mappers/check-activities')
+
+function ViewModel (cdo, activities, backNav) {
+  const mappedActivities = activities.map(mapActivityDtoToCheckActivityRow)
+
+  this.model = {
+    backLink: backNav.backLink,
+    srcHashParam: backNav.srcHashParam,
+    dog: {
```

Comment:
I was planning to keep the event generic by retaining 'pk' so that we could use the same structure for owner activities. Pk would be. a personReference if source was 'owner' but indexNumber if source was 'dog'

Reply:
sounds good, have made this ``indexNumber: cdo.dog.indexNumber,`` for now, and can iterate on next story

Reply:
```
this.model = {
    backLink: backNav.backLink,
    srcHashParam: backNav.srcHashParam,
    pk: cdo.pk,
    source: cdo.source,
    events: mappedActivities
  }
```